tap_universal_file/tap.py

```diff
@@ -291,6 +294,14 @@ class TapUniversalFile(Tap):
                 "It is recommended that you do not modify this setting."
             ),
         ),
+        th.Property(
+            "schema_file",
+            th.StringType,
+            default="",
+            description=(
+                "Schema file to override the discovered schema. It should be Inside the schemas folder"  # noqa: E501
```

Comment:
We should point to a file relative to the current working directory, and also accept an absolute file path